deps/rabbit/src/rabbit_stream_coordinator.hrl

```diff
@@ -48,16 +48,21 @@
                  conf :: osiris:config(),
                  nodes :: [node()],
                  members = #{} :: #{node() := #member{}},
-                 listeners = #{} :: #{pid() := LeaderPid :: pid()},
+                 listeners = #{} :: #{pid() | %% v0 & v1
+                                      {pid(), leader | member} %% v2
+                                      := LeaderPid :: pid()} | {node(), LocalPid :: pid()},
```

Comment:
Why include a `node()` here? wont the node always be the one of the listener pid?